### PR TITLE
Add QueueEmitEncounter to SuggestEditHP callback

### LINKED ISSUE #137: Accepting a player damage suggestion does not update player view
Missing a call to `QueueEmitEncounter`

### PR DESCRIPTION
**Pull request by dlom**
Fixes #137

### CODE REVIEW

File: client/Commands/CombatantCommander.ts [hunk 1]
```diff
@@ -145,8 +145,10 @@ module ImprovedInitiative {
                 return false;
             }
 
-            const combatantNames = suggestedCombatants.map(c => c.Name()).join(', ');
-            const prompt = new AcceptDamagePrompt(suggestedCombatants, suggestedDamage, suggester, this.tracker.EventLog.LogHPChange);
```
Comment (cynicaloptimist) [inline client/Commands/CombatantCommander.ts]: Have this parameter just take the whole tracker object so this behavior can be moved into the AcceptDamagePrompt class.

**Comment by cynicaloptimist** [comment]
Also increment the patch version in package.json to 0.11.1.

**Comment by cynicaloptimist** [comment]
Thanks Mark! I'll deploy this later tonight.

**Comment by dlom** [comment]
No problem.  Sorry to bother with all these PRs; there are a lot of moving parts to this feature :)

**Comment by cynicaloptimist** [comment]
Not a bother at all! I appreciate you verifying my changes, it would be simpler if my "refactorings" didn't break your feature in the first place. :-)

**Pull request merged**